.expeditor/update_dep.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -evx
+
+branch="expeditor/${GEM_NAME}_${VERSION}"
+git checkout -b "$branch"
+
+bundle update $GEM_NAME
```

Comment:
Could add `-v $VERSION`. That could potentially highlight if there is some promotion problem later down the line

Reply:
Do you mean a conflict that prevents pulling in latest?